### ISSUE TITLE
Trying to integrate TFLite as a Frame Processor Plugin❓ 

### ISSUE THREAD

**Issue by iyinoluwamt** [cuvent/react-native-vision-camera#1073]
### Question

Any tips on how to do this and/or structure this project

### What I tried

I've tried to go off the example project directory but I'm not sure how to implement TFLite in the ExampleProcessorPlugin code

### VisionCamera Version

2.13.3

### Additional information

- [ ] I am using Expo
- [X] I have read the [Troubleshooting Guide](https://mrousavy.github.io/react-native-vision-camera/docs/guides/troubleshooting)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for similar questions [in the issues page](https://github.com/mrousavy/react-native-vision-camera/issues?q=is%3Aopen+is%3Aissue+label%3A%22❓+question%22) as well as [in the discussions page](https://github.com/mrousavy/react-native-vision-camera/discussions) and found none.

**Comment by mrousavy** [comment]
Hey, sorry I've never used TFLite before - what exactly are you struggling with? What did you try?

**Comment by felixdittrich92** [comment]
Hi @iyinoluwamt,

you could convert your model into ONNX format and use:
(ensure to pass your dummy input batch size for TF models as None to provide different batch sizes otherwise this will be a fixed value ([tf2onnx](https://github.com/onnx/tensorflow-onnx) does not provide dynamic axes like PyTorch))
react-native: https://www.npmjs.com/package/onnxruntime-react-native
or native: https://onnxruntime.ai/docs/get-started/with-java.html   java and objective-c bindings exists

**Comment by iyinoluwamt** [comment]
> Hey, sorry I've never used TFLite before - what exactly are you struggling with? What did you try?

These are from my Swift Frame Processor Plugin file. I tried to model it off of "https://github.com/tensorflow/examples/blob/master/lite/examples/object_detection/ios/README.md".

<img width="468" alt="Run" src="https://user-images.githubusercontent.com/96902046/172645936-77e6ea25-1f37-4111-ab54-14bff87c4401.png">
<img width="435" alt="Inference" src="https://user-images.githubusercontent.com/96902046/172645951-ab8a0fed-3697-407a-97df-c9a65c5f637b.png">

<img width="468" alt="babel config" src="https://user-images.githubusercontent.com/96902046/172646860-50e21e10-c4d9-475c-af0a-9bd97c2830f3.png">
<img width="468" alt="Picture1" src="https://user-images.githubusercontent.com/96902046/172646810-773d1e1f-114f-47d5-89f1-ee6dbbbe5faa.png">
<img width="468" alt="FrameProcessor js" src="https://user-images.githubusercontent.com/96902046/172646828-f6417cc1-1f7e-4486-ab46-805d40c14cc6.png">

My issue is that this returns `[]` on all frames so I'm not sure why no inferences are being made (TFLite has a working Swift example). I can provide more if you need.

**Comment by iyinoluwamt** [comment]
> Hi @iyinoluwamt,
> 
> you could convert your model into ONNX format and use: (ensure to pass your dummy input batch size for TF models as None to provide different batch sizes otherwise this will be a fixed value ([tf2onnx](https://github.com/onnx/tensorflow-onnx) does not provide dynamic axes like PyTorch)) react-native: https://www.npmjs.com/package/onnxruntime-react-native or native: https://onnxruntime.ai/docs/get-started/with-java.html java and objective-c bindings exists

Do you have any idea how I could get the frame by frame input for the ONNX model without using native code and just react-native libraries?

**Comment by mrousavy** [comment]
See https://github.com/tensorflow/tfjs/issues/7773 :)

**Comment by mrousavy** [comment]
Hey - I built a fast C++ / JSI / GPU-accelerated plugin just for this: https://github.com/mrousavy/react-native-fast-tflite 🥳
